x-pack/plugins/security_solution/public/management/cypress/e2e/mocked_data/artifact_tabs_in_policy_details.cy.ts

```diff
@@ -81,12 +81,12 @@ describe('Artifact tabs in Policy Details page', () => {
   });
 
   for (const testData of getArtifactsListTestsData()) {
-    beforeEach(() => {
-      login();
-      removeExceptionsList(testData.createRequestBody.list_id);
-    });
-
     describe(`${testData.title} tab`, () => {
+      beforeEach(() => {
+        login();
+        removeExceptionsList(testData.createRequestBody.list_id);
```

Comment:
Maybe we could move it to the `afterEach`, so we know that we don't leave the lists created?

Reply:
It's in both places:
- in `beforeEach` to make sure other tests don't mess up this test (but only for the given artifact list to make tests faster, also for re-running them locally),
- in `after`, by calling `removeAllArtifacts()` which deletes all lists